libtrace/src/frame_arch.h

```diff
@@ -385,6 +385,42 @@ enum frame_architecture
   frame_arch_aarch64,   /* AArch64.  */
 #define frame_mach_aarch64 0
   frame_arch_8051,/* Intel 8051. */
+  frame_arch_hexagon, /* Qualcomm DSP architecture Hexagon */
```

Comment:
Please put this after the `frame_arch_sm83` (as well as in `frame_arch.ml`) so compatibility with previous commits on master is preserved.